addons/account/models/company.py

```diff
@@ -66,6 +69,8 @@ class ResCompany(models.Model):
     account_setup_fy_data_done = fields.Boolean('Financial Year Setup Marked As Done', help="Technical field holding the status of the financial year setup step.")
     account_setup_coa_done = fields.Boolean(string='Chart of Account Checked', help="Technical field holding the status of the chart of account setup step.")
     account_setup_bar_closed = fields.Boolean(string='Setup Bar Closed', help="Technical field set to True when setup bar has been closed by the user.")
+    invoice_reference_type = fields.Selection(string='Default Invoice Reference Type', selection='_get_invoice_reference_types',
+                                              default='none', help='Default Reference Type on Invoices.')
```

Comment:
this is an error use a method to fill the types, when we have the add option available in the new api don's you think?